mysql-test/suite/archive/archive.test

```diff
@@ -6,6 +6,7 @@
 
 CALL mtr.add_suppression("Unsafe statement written to the binary log using statement format since BINLOG_FORMAT = STATEMENT");
 call mtr.add_suppression("Table 't1' is marked as crashed and should be repaired");
+call mtr.add_suppression("Table './test/t1' is marked as crashed and should be repaired");
```

Comment:
Please ensure that we do not use '/' in the suppression.  (Will not work on windows)